src/selectors/event.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+import type { Event } from "../data/event";
+
+const locale = "en-GB";
```

Comment:
You can import this from `/src/data/locale.js`.

Reply:
Shows how old this PR is...